travis/travis_install_nightly

```diff
@@ -91,13 +112,17 @@ echo "Install webkit (wkhtmltopdf) patched"
 #     |___ <OdooRepo>-<Branch>/         <-- Odoo Server
 #     |___ maintainer-quality-tools/
 #     |___ build/<Owner>/<TestedRepo>/
-#     |___ dependencies/<DependencyRepo1>/
-#     |___ dependencies/<DependencyRepo2>/
```

Comment:
These dependency dirs will still be there if we have an `oca_dependencies.txt`, so IMO we should keep this.